README.md

```diff
@@ -0,0 +1,32 @@
+# KTorrent (org.kde.ktorrent)
+
+[Homepage](https://kde.org/applications/ktorrent) |
+[User documentation](https://docs.kde.org/stable5/en/extragear-network/ktorrent/index.html) |
+[Sources](https://invent.kde.org/network/ktorrent)
+
+## Reporting bugs
```

Comment:
Github issue template?

Reply:
That's a good idea. Maybe we should do that for all the KDE repos?

Reply:
https://invent.kde.org/teams/flathub/issues/-/issues/4#note_119906